config/locales/en/activemodel.yml

```diff
@@ -491,6 +491,16 @@ en:
               file_virus: "%{file_name} contains a virus"
               no_file_chosen: Upload your client's bank statements
               system_down: There was a problem uploading your file - try again
+        providers/means/housing_benefit_form:
+          attributes:
+            housing_benefit:
+              inclusion: Select one of the options
```

Comment:
Not sure if this has been content checked but I think usually for yes/no radio button pages, we have the none selected error message as something more informative like 'Select if your client receives housing benefit' or 'Select yes if your client receives housing benefits' - maybe one to check with Chris?